codegen/Cargo.toml

```diff
@@ -9,34 +9,34 @@ version = "0.2.3"
 
 [build-dependencies.serde_codegen]
 optional = true
-version = "0.7.6"
+version = "=0.7.6"
```

Comment:
This version of the crate was yanked, I wonder if we have code dependent on that broken behavior.